h/streamer/websocket.py

```diff
@@ -18,8 +18,28 @@
 # below.
 MESSAGE_HANDLERS = {}
 
+
 # An incoming message from a WebSocket client.
-Message = namedtuple('Message', ['socket', 'payload'])
+class Message(namedtuple('Message', [
+    'socket',
+    'payload',
+])):
+    def reply(self, payload, ok=True):
```

Comment:
I'm not sure about this. To me having `{"error": false}` in a successful message back to the client (hopefully the 90% case) is more likely to cause confusion than `{"ok": true}`, and I think the kwarg here should correspond directly to what we send to the client.

(Not to mention the fact that the `error` key is already in use and contains the actual error data.)